tests/test_checkout.py

```diff
@@ -1,18 +1,66 @@
 import pytest
 from django.conf import settings
 from django.contrib.auth.models import AnonymousUser
-from mock import MagicMock, Mock
+from django.forms import model_to_dict
+from mock import Mock
 from prices import Price
+from satchless.item import partition
 
+from saleor.cart.models import ProductGroup, Cart, CartLine
 from saleor.checkout import views
 from saleor.checkout.core import STORAGE_SESSION_KEY, Checkout
 from saleor.shipping.models import ShippingMethodCountry
 from saleor.userprofile.models import Address
 
 
-def test_checkout_version():
-    checkout = Checkout(Mock(), AnonymousUser(), 'tracking_code')
-    storage = checkout.for_storage()
+@pytest.fixture
+def anonymous_checkout():
+    return Checkout(Mock(), AnonymousUser(), 'tracking_code')
+
+
+@pytest.fixture
+def cart_with_partition_factory(cart, monkeypatch, product_in_stock):
+    def generate_cart(items):
+        partitions = []
+        for item in items:
+            is_shipping_required = item.get('is_shipping_required', True)
+            money_function_mock = Mock(return_value=Price(item['cost'],
```

Comment:
Please break the line after the opening parenthesis. Keeping these aligned is a major PITA and there's no obvious way to increase indentation if you open another parenthesis.